app/src/main/res/layout/list_patient_search_header.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<TextView android:id="@+id/patientsearch_header"
```

Comment:
We are following the convention of using `textAppearance` for styling attributes related to text rendering. Can you do the same here?

Reply:
done